knx.py

```diff
@@ -14,6 +14,8 @@
 KNXWRITE = 0x80
 KNXREAD = 0x00
 
+DT_BIT = '1.*'
```

Comment:
Yes it is the standardized types def. There is other types def but I didn't had time to add this :(
Ok for tests, I will had some (from my prod running ones ;) )